runtime/test_artifacts/generator.go

```diff
@@ -155,8 +130,7 @@ func NewAccounts(n int, privateKeys ...string) (accounts []modules.Account) {
 }
 
 // TODO: The current implementation of NewActors  will have overlapping `ServiceUrl` for different
-//
-//	types of actors which needs to be fixed.
+//       types of actors which needs to be fixed.
```

Comment:
micro NIT

Reply:
I think this line is correct, it means that accidentally merged the wrong one (which has an empty line in between). 

Here I am comparing with the commit from 2 weeks ago:

![image](https://user-images.githubusercontent.com/29378614/199823305-dc27cc4e-1284-4b83-bb1c-7802a21ca67a.png)

https://github.com/pokt-network/pocket/commit/cc5beb5#diff-ee06bf613911836fdf48424397b7ec556c2e1a8ddba5adecca4c1782aea24087R148-R149

Reply:
I must've been looking at a weird commit. Ignore.